src/main/scala/devices/tilelink/Plic.scala

```diff
@@ -308,18 +308,18 @@ class TLPLIC(params: PLICParams, beatBytes: Int)(implicit p: Parameters) extends
     node.regmap((priorityRegFields ++ pendingRegFields ++ enableRegFields ++ hartRegFields):_*)
 
     if (nDevices >= 2) {
-      val claimed = claimer(0) && maxDevs(0) > 0
+      val claimed = claimer(0) && maxDevs(0) > 0.U
       val completed = completer(0)
       property.cover(claimed && RegEnable(claimed, false.B, claimed || completed), "TWO_CLAIMS", "two claims with no intervening complete")
       property.cover(completed && RegEnable(completed, false.B, claimed || completed), "TWO_COMPLETES", "two completes with no intervening claim")
 
       val ep = enables(0).asUInt & pending.asUInt
       val ep2 = RegNext(ep)
-      val diff = ep & ~ep2
-      property.cover((diff & (diff - 1)) =/= 0, "TWO_INTS_PENDING", "two enabled interrupts became pending on same cycle")
+      val diff = ep & (~ep2).asUInt
```

Comment:
```suggestion
      val diff = ep & ~ep2
```
Don't change this, none relate to chisel3 port